src/components/NewsletterForm/index.tsx

```diff
@@ -56,15 +56,16 @@ export const NewsletterForm = ({ className = '', placement }: NewsletterFormProp
                     <>
                         <div className="float-right md:hidden -mt-6 2xs:-mt-12">
                             <CloudinaryImage
-                                src="https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/src/images/swole-hog.png"
+                                src="https://res.cloudinary.com/dmukukwp6/image/upload/engineer_47d6638eae.png"
                                 objectFit="contain"
                                 className="w-full h-full max-w-[170px]"
                             />
                         </div>
                         <p className="!text-sm opacity-50 !m-0">Subscribe to our newsletter</p>
                         <h4 className="relative !text-2xl !m-0 !leading-tight">Product for Engineers</h4>
                         <p className="!m-0 !text-sm md:!text-[15px] !leading-normal !pt-1">
-                            Helping engineers and founders flex their product muscles
+                            Sharing what we learn about building successful products. Read by 25,000+ founders and
+                            developers.
```

Comment:
Too much text IMO. Maybe we just drop the first sentence. I dont think we need to explain in too much detail what it is
```suggestion
                            Read by 25,000+ founders and
                            developers.
```

Reply:
I like the social proof of 25k readers btw